theano/gpuarray/corr3d_gemm.c

```diff
@@ -35,14 +35,18 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 // GPU kernel for the case of dilation
 KERNEL void dilated_im3d2col_kernel(const ga_size n,
     GLOBAL_MEM const DTYPE_INPUT_0 * data_im,
+    const ga_size offset_im,
     const ga_size data_im_offset,
```

Comment:
As discussed IRL, it would be nice to have a comment, for instance something like
"`offset_im` corresponds to the data pointer offset of the GpuArray object, whereas `data_im_offset` is an offset of elements inside the array".